### ISSUE TITLE
Integrate countdown inside the player instead of static message ouside of the player when autoplay is enabled

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#1214]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
I noticed that there is a static message ("Playing next video in 5 seconds. Click to cancel.") when waiting for the next video to appear. I think its better to let the user know how much time they have left instead of just guessing when the 5sec will be over. 

This will enhance user experience in 2 ways:
1. Users will exactly know how much time is left when the new video starts. 
2. Users will exactly know how much time is left to prevent the next video from playing. 

**Describe the solution you'd like to see implemented**
Integrate the countdown into the player.

This is because there are 3 flaws in the current implementation:
1. When fullscreen is enabled u cant see the message appear.
2. When Fullwindow is enabled u cant see the message appear.
3. When PIP is enabled u cant see the message appear.

**Comment by PikachuEXE** [comment]
I tried to take a stab but the result is not good...

https://user-images.githubusercontent.com/1018543/115650319-93aef400-a35b-11eb-9892-5cebb4a7e79b.mp4

Code change:
https://github.com/FreeTubeApp/FreeTube/compare/development...PikachuEXE:feature/playlist-autoplay-countdown

**Comment by GilgusMaximus** [comment]
The idea is good, I am not so sure however that it will work well because these toasts, as you might have noticed get pushed to the top. Maybe you could make the toast less than 1 second, so that it disappears before the next one occurs. Then the previous one does not get pushed upwards

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I think i worded wrong what i mean with integrate countdown into the player, because this implementation still occurs on the outside of the player. To clear things up i screen recored YT implementation.

https://user-images.githubusercontent.com/73130443/115767310-83693a00-a398-11eb-8ba0-0dd6fe5d361b.mp4

There has to be a countdown and cancel button. The other things like the preview of the next video and the play now button are optional.

**Comment by GilgusMaximus** [comment]
Yeah, I mean @PikachuEXE's approach is an easy approach which solves the problem with existing functionality. For the moment that would be fine. But in the long run, something like on YouTube will obviously be more pleasant to look at

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hmm i dont fully agree with u because of the flaws i pointed out in my issue. The proposed implementation of Pikachu is still outside of the player. This implementation wont fix the 3 flaws i pointed out.

**Comment by PikachuEXE** [comment]
Adjusted the time for the messages to ensure there is no flashing effect (not sure the actual cause)

https://user-images.githubusercontent.com/1018543/116199247-7d44d600-a769-11eb-9790-c4a394d0d361.mp4

**Comment by GilgusMaximus** [comment]
> 
> 
> Hmm i dont think i can agree with u because of the flaws i pointed out in my issue. The proposed implementation of Pikachu is still outside of the player. This implementation wont fix the 3 flaws i pointed out. It is a requirement to implement it into to player to fix the flaws, so its not only for the looks.

True that. But it is still an improvement over the current system. So I'd leave this open, even with @PikachuEXE changes

**Comment by PikachuEXE** [comment]
https://github.com/FreeTubeApp/FreeTube/pull/1234 opened

**Comment by Svallinn** [comment]
That's my bad.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE just noticed that the countdown continues to appear when i leave the video page. So lets say the countdown started and i go into the settings page i will see the countdown.

**Comment by PikachuEXE** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc 
Should be fixed with #1287

**Comment by PikachuEXE** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc 
Should be fixed with #1287
